app/views/product_users/index.html.haml

```diff
@@ -20,6 +20,12 @@
   %p.notice= text("none")
 - else
   - if @product.has_product_access_groups?
+    = form_tag facility_instrument_users_path(format: :csv), method: :get, class: "search_form" do
+      = hidden_field_tag :email, current_user.email, disabled: true
+      = hidden_field_tag :format, params[:format], disabled: true
+
+    = link_to t("reports.instruments.product_users.export"), url_for(format: :csv), class: "js--exportSearchResults pull-right", data: { form: ".search_form" }
```

Comment:
Let's keep this simple and just send the email to the current user instead of prompting.

Reply:
I thought this was the consistent way of doing things for the other Export to CSV links that I saw? Also, as an admin might I want to send the file to someone else?

Reply:
Good points, let's keep the prompt.